### PR TITLE
[jsbn] create jsbn package

### PR DESCRIPTION
**Pull request by upgradingdave**
New package:

**Extern:** Generated.

This includes both jsbn and jsbn2. I wrote a custom deps.clj to include both namespaces. These are both dependencies for AWS Cognito Identity SDK which I'll submit a separate pull request for shortly.

### CODE REVIEW

File: jsbn/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cljsjs/jsbn
+
+[](dependency)
+```clojure
+[cljsjs/jsbn "0.1.1-1"] ;; latest release
```
Comment (miikka) [inline jsbn/README.md]: `+lib-version+` is `0.1.0` but it says `0.1.1` here. (If you don't fix it, I believe @deraen-robot does it automatically.)

**Comment by miikka** [comment]
Looks good, although what I said about extra stuff in #918 applies here as well. 👍

File: jsbn/build.boot [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   (download :url "http://www-cs-students.stanford.edu/~tjw/jsbn/jsbn.js"
+             :checksum "5E6FE0DA9EF45687E52781B9A646454E"
+             :unzip false)
+
+   (download :url "http://www-cs-students.stanford.edu/~tjw/jsbn/jsbn2.js"
+             :checksum "402CB831E9747F7B0029CE06B8CD76BD"
+             :unzip false)
+
+   (sift :move {#"^jsbn.js" "cljsjs/jsbn/development/jsbn.inc.js"
+                #"^jsbn.js" "cljsjs/jsbn/production/jsbn.min.inc.js"
```
Comment (Deraen) [inline jsbn/build.boot]: There is no need to separate development and minified file if the minified file is not available.

You can just copy the file to `cljsjs/jsbn/development/jsbn.inc.js` or `cljsjs/jsbn/common/jsbn.inc.js` and only define `:file` in deps.cljs.

Comment (Deraen) [inline jsbn/build.boot]: And you already only define `:file` in deps.cljs so the minified files here are unncessary.

**Comment by upgradingdave** [comment]
Thanks guys, I'll clean this up asap.

**Comment by Deraen** [comment]
Closing due to inactivity. Please reopen if you want to finish this PR.

**Pull request closed without merging**